.github/workflows/integration_test.yml

```diff
@@ -24,29 +24,29 @@ jobs:
       - name: install libbpf
         uses: sustainable-computing-io/kepler-action@v0.0.3
         with:
-          ebpfprovider: bcc
+          ebpfprovider: libbpf
       # build kepler image
       - name: build and export Kepler image
         run: |
           make build_containerized
```

Comment:
shouldn't we set the ATTACHER=libbpf for all `make` invocations?

Reply:
i do not want  to make too many changes at once.

Reply:
let me rephrase ... currently ` bcc` is the default attach method because this PR doesn't modify Makefile. So, for all make invocations that builds kepler or has dependency on `ATTACHER_TAG` (directly or indirectly), shouldn't we explicitly set `ATTACHER_TAG=libbpf` ?

Reply:
i just changed the name of the workflow here. if correct images were built, that should remain same.

Reply:
Resolving this, since the original libbpf test did not require explicit ATTACHER_TAG